.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```

Comment:
use same config as the one provided my microverse [[see config here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)]